app/views/users/_tabs.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="row">
+  <div class="col md-6 offset-3">
```

Comment:
Tabs are off-center & don't align with the page title

Reply:
fixed tabs alignment